CHANGELOG.md

```diff
@@ -4,6 +4,73 @@ All notable changes to this project will be documented in this file.
+* InstanceType default is now `t3.large` (was `t2.nano`) [#699](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/699) ([pda](https://github.com/pda))
+* Made ECR hook an `environment` hook (was `pre-command`). [#677](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/677) ([pda](https://github.com/pda))
+* Mappings file format has changed to list both Linux and Windows AMIs [#569](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/569) ([lox](https://github.com/lox))
+* We now warn instead of hard-fail when there's no configured SSH keys [#669](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/669) ([pda](https://github.com/pda))
+* We now only set git-mirrors-path when EnableAgentGitMirrorsExperiment is set [#698](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/698) ([pda](https://github.com/pda))
+* Set RootVolumeName appropriately and allow it to be overridden [#593](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/593) ([jeremiahsnapp](https://github.com/jeremiahsnapp))
+
+### Fixed
+* Ensure terminate-instance exits gracefully [#707](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/707) ([patrobinson](https://github.com/patrobinson))
+* Stop trying to call poweroff after the agent shuts down [#728](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/728) ([yob](https://github.com/yob))
+* Update agent config to use `tags-from-ec2-meta-data` [#727](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/727) ([yob](https://github.com/yob))
+* Set correct content-type on YAML template files shipped to S3 [#683](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/683) ([kyledecot](https://github.com/kyledecot))
+* Windows: pin aws-cli to v1; v2 breaks 'aws ecr get-login' [#670](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/670) ([pda](https://github.com/pda))
+* Windows: Pin git-for-windows to v2.22.0 [#632](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/632) ([jeremiahsnapp](https://github.com/jeremiahsnapp))
+* Windows: Add default InstanceCreationTimeout value [#595](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/595) ([jeremiahsnapp](https://github.com/jeremiahsnapp))
+* Windows: Ensure random password satisfies policy [#628](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/628) ([jeremiahsnapp](https://github.com/jeremiahsnapp))
+* Windows: Ensure unhealthy instances get marked correctly [#614](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/614) ([jeremiahsnapp](https://github.com/jeremiahsnapp))
+* Windows: run the bootstrap script AFTER the buildkite-agent user is created [#600](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/600) ([tduffield](https://github.com/tduffield))
+* Windows: Ensure powershell scripts fail when executable fails [#633](https://github.com/buildkite/elastic-ci-stack-for-aws/pull/633) ([jeremiahsnapp](https://github.com/jeremiahsnapp))
```

Comment:
As above, these windows lines feel like an implementation detail of the headline windows feature, maybe we can drop them to keep the signal high?

Reply:
I'll roll these up into the windows announcement at the top, but I definitely want to acknowledge work put in to stabilise windows support